tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
```

Comment:
If we already define tox can we reuse the scaffolding template that we use in all our services?

Reply:
I believe we agreed that this can be done later (ISV-919).